lib/render.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const _ = require('lodash'),
+  clayUtils = require('clayutils'),
+  mediaService = require('./media');
+
+let log = require('./log').setup({ file: __filename }),
+  amphoraFs = require('amphora-fs'),
+  hbs;
+
+/**
+ * Log the render time
+ *
+ * @param  {Object} hrStart
+ * @param  {String} msg
+ * @param  {String} route
+ * @return {Function}
+ */
+function logTime(hrStart, msg, route) {
+  return () => {
+    const diff = process.hrtime(hrStart),
+      ms = Math.floor((diff[0] * 1e9 + diff[1]) / 1000000);
+
+    log('info', `${msg}: ${route} (${ms}ms)`, {
+      renderTime: ms,
+      route,
+      type: 'html'
+    });
+  };
+}
+
+/**
+ * Construct the object that is needed by Kiln for rendering in
+ * edit mode. Requires reading from the file system.
+ *
+ * @param  {Object} data
+ * @param  {Object} meta
+ * @return {Object}
+ */
+function makeState(data, meta) {
```

Comment:
we should talk about how we want to handle edit mode for amp.... i think we should make it unavailable?

Reply:
Oh yeah, I have been trying to remove anything related to edit mode with this AMP renderer.  I  need to adjust this comment to not reference edit mode, although I think the few things that I left here in makeState are necessary downstream for building up the HTML string.